src/Model/MyParcelRequest.php

```diff
@@ -98,16 +98,22 @@ public function getError()
      * @param string $body
      * @param string $requestHeader
      *
+     * @param bool   $sendMail
+     *
      * @return $this
      */
-    public function setRequestParameters($apiKey, $body, $requestHeader): MyParcelRequest
+    public function setRequestParameters(string $apiKey, string $body, string $requestHeader, bool $sendMail = true): MyParcelRequest
```

Comment:
`bool $sendMail = true` is heel specifiek voor jouw return situatie, maar totaal niet relevant voor andere requests. Ik zou hier een array doen met met $query = []

Het aanroepen kan dan:
```            ->setUserAgent($this->getUserAgent())
            ->setRequestParameters(
                $apiKey,
                $data,
                MyParcelRequest::REQUEST_HEADER_RETURN
                MyParcelRequest::REQUEST_HEADER_RETURN,
                ['send_return_mail' => 1]
            )
```

Reply:
aangepast, send_return_mail zal nu als url parameter worden verstuurd